### PR TITLE
Create "Connectivity / benthos-umh" page

### PR DESCRIPTION
**Pull request by DaruZero**
# Description

This PR adds the Feature page for Benthos UMH

## Related issues

Closes https://github.com/united-manufacturing-hub/MgmtIssues/issues/1055

## Checklist

*You can also fill these out after creating the PR. If you're unsure about any of them, don't hesitate to ask. We're here to help! This is simply a reminder of what we are going to look for before merging your code.*

- [x] I have read the [contributing guidelines](https://umh.docs.umh.app/docs/development/contribute/new-content/add-documentation/) and the [code of conduct](CODE_OF_CONDUCT.md).
- [x] I have followed the [documentation](https://umh.docs.umh.app/docs/development/contribute/documentation/style/) style.

### CODE REVIEW

File: umh.docs.umh.app/content/en/docs/features/connectivity/benthos-umh.md [hunk 1]
```diff
@@ -1,7 +1,122 @@
...
+with IT tools is necessary to stream data from an OPC UA server. With Benthos
+UMH, you can easily connect to an OPC UA server, define the nodes you want to
+stream, and send the data to the Unified Namespace.
+
+Additionally, Benthos has proven more reliable than other tools, like Node-RED,
+when it comes to handling large amounts of data.
+
+## What can I do with it?
+
+Benthos UMH offers some benefits over mainline Benthos, including:
```
Comment (JeremyTheocharis) [inline umh.docs.umh.app/content/en/docs/features/connectivity/benthos-umh.md:35]: offers benefits (without over mainline benthos, as also mainline benthos is open source, etc.)

File: umh.docs.umh.app/content/en/docs/features/connectivity/benthos-umh.md [hunk 2]
```diff
@@ -1,7 +1,122 @@
...
+      site: aachen
+      area: packaging
+      line: packaging_1
+      workcell: blister
+      originID: PLC13
+      tagName: machineState
+      useCase: _historian
+```
+
+Mandatory fields are `opcuaID`, `enterprise`, `tagName` and `useCase`. Read
```
Comment (JeremyTheocharis) [inline umh.docs.umh.app/content/en/docs/features/connectivity/benthos-umh.md:90]: I think these are not described in the datamodel. enterprise, site, area are described as part of the ISA-95 model, usecase also belings there, but opcuaID and tagName are not (explitely) described.

Mandatory fields are .... opcuaID is the NodeID in OPC-UA and can also be a folder (see benthos README for more information). The remaining components are components of the resulting topic / ISA-95 structure (see also our datamodel). By default, the schema will always be in _historian, and tagName is the keyname.

**Review by JeremyTheocharis — CHANGES_REQUESTED** [review]

File: umh.docs.umh.app/content/en/docs/features/connectivity/benthos-umh.md (hunk 2 above)
Comment (Ayato77) [reply umh.docs.umh.app/content/en/docs/features/connectivity/benthos-umh.md]: updated

File: umh.docs.umh.app/content/en/docs/features/connectivity/benthos-umh.md (hunk 1 above)
Comment (Ayato77) [reply umh.docs.umh.app/content/en/docs/features/connectivity/benthos-umh.md]: updated

File: umh.docs.umh.app/content/en/docs/features/connectivity/benthos-umh.md [hunk 3]
```diff
@@ -1,7 +1,124 @@
...
+Unified Namespace.
+
+## When should I use it?
+
+OPC UA is a communication protocol coming from the OT industry, so integration
+with IT tools is necessary to stream data from an OPC UA server. With Benthos
+UMH, you can easily connect to an OPC UA server, define the nodes you want to
+stream, and send the data to the Unified Namespace.
+
+Additionally, Benthos has proven more reliable than other tools, like Node-RED,
```
Comment (JeremyTheocharis) [inline umh.docs.umh.app/content/en/docs/features/connectivity/benthos-umh.md:30]: *in our tests. see also https://learn.umh.app/blog/our-open-source-docker-container-to-connect-opc-ua-with-the-unified-namespace/#testing-existing-solutions

**Review by JeremyTheocharis — CHANGES_REQUESTED** [review]

File: umh.docs.umh.app/content/en/docs/features/connectivity/benthos-umh.md [hunk 4]
```diff
@@ -1,7 +1,125 @@
...
+when it comes to handling large amounts of data in our tests. See also 
+[this article](https://learn.umh.app/blog/our-open-source-docker-container-to-connect-opc-ua-with-the-unified-namespace/#testing-existing-solutions).
```
Comment (DaruZero) [inline umh.docs.umh.app/content/en/docs/features/connectivity/benthos-umh.md:32]: Word it like this:

```markdown
Furthermore, [in our tests](https://learn.umh.app/blog/our-open-source-docker-container-to-connect-opc-ua-with-the-unified-namespace/#testing-existing-solutions),
Benthos has proven more reliable than tools like Node-RED, when it comes
to handling large amounts of data.
```

Comment (Ayato77) [reply umh.docs.umh.app/content/en/docs/features/connectivity/benthos-umh.md]: replaced

**Review by JeremyTheocharis — APPROVED** [review]

**Pull request merged**